lib/elixir/pages/Naming Conventions.md

```diff
@@ -105,4 +105,4 @@ When you see `length`, the operation runs in linear time ("O(n) time") because t
 
 Examples: `Kernel.length/1`, `String.length/1`
 
-In other words, `size` functions will take the same amount of time whether the data structure is tiny or huge. `length` functions will take more time as the data structure grows in size.
+In other words, _size_ functions will take the same amount of time whether the data structure is tiny or huge. _length_ functions will take more time as the data structure grows in size.
```

Comment:
@josevalim I think we should not use backticks here, since that will mean that a function is named exactly "size" or "length", when the message it tries to convey is that the word size/length may be used in the function name.
So my suggestions are:
- use italics
- use bold
- use italic and bold
- rephrase it:  such as, "In other words, functions using the word "size" in its name will take the same amount of time whether the data structure is tiny or huge. Conversely, functions having "length" in its name will take more time as the data structure grows in size."